pkg/observable/errors.go

```diff
@@ -1,8 +1,12 @@
 package observable
 
-import errorsmod "cosmossdk.io/errors"
+import (
+	skderrors "cosmossdk.io/errors"
+)
```

Comment:
Are these import reformats (one- to multi-line) a linter thing or a preference?

Reply:
It would be my linter doing this - I am unsure of standards around single line imports but my linter doesn't like them apparently.